css/style.css

```diff
@@ -13,4 +13,18 @@
 ul.pagination {
 	display: flex;
 	justify-content: center;
+}
+
+.brand-logo {
+    white-space: nowrap;
+}
+
+.container {
+    overflow-wrap: anywhere;
+}
+
+@media only screen and (max-width: 600px) {
+    .container {
+        width: 100%;
```

Comment:
Weirdly, but default in materialize for small screen media is 90%. In real, 10% of screen width on small display is actualy **huge** waste of space